ega-charts/localega/templates/doa-deploy.yaml

```diff
@@ -89,7 +89,7 @@ spec:
         - name: CERT_KEY
           value: "/ega/tls/doa.ca.key.der"
         - name: CRYPT4GH_PRIVATE_KEY_PATH
-          value: "/etc/ega/c4gh/key.1.c4gh.sec"
+          value: "/etc/ega/c4gh/ega_key.c4gh.sec"
```

Comment:
this is more of a question, I understand using it like this for e2e-tester, but why for DOA ?
wouldn't it be a bit better to have in a secret, I mean the `lega-secret` it is already there
the same goes for the key.pass below.

Reply:
It is in a secret, that's just the path tho the file to read.